critiquebrainz/db/review.py

```diff
@@ -19,6 +19,7 @@
     "place",
     "release_group",
 ]
+RATING_SCALE_1_5 = {20: 1, 40: 2, 60: 3, 80: 4, 100: 5}
```

Comment:
Since there is a module specifically related to ratings it might be better to put these values there instead.

Reply:
Sorry, didn't get you. Which module are you talking about?

Reply:
I guess there's no `rating` module there, but https://github.com/metabrainz/critiquebrainz/blob/master/critiquebrainz/db/avg_rating.py is probably the closes. In any case, a more relevant place to have these definitions.

Reply:
Will `__init__.py` be a good place for this? I know avg_rating.py is related to ratings but it doesn't seem a good fit for rating conversion scales.

Reply:
Sure, not ideal but better.